src/view/wallet/Settings.svelte

```diff
@@ -0,0 +1,25 @@
+<script lang="ts">
+    import type {NanoAccount} from "../../machinery/models";
+    import Seperator from "../../components/Seperator.svelte";
+    import LabelledInput from "../../components/LabelledInput.svelte";
+    import Button from "../../components/Button.svelte";
+
+    export let account: NanoAccount;
+    export let storeFunction: () => void
+
+    let aliasValue: string = account?.alias
+
+    const setAlias = (event) => {
+        aliasValue = event.target.value;
+    }
+
+    const save = async () => {
+        account.alias = aliasValue
+        storeFunction()
+    }
+
+</script>
+
+<Seperator languageId="account-settings" />
+<LabelledInput languageId="account-alias" on:change={setAlias} bind:value={aliasValue}></LabelledInput>
```

Comment:
You probably have to remove the previous value as value. Probably better to use it as placeholder.
When I go into the input field I am not able to delete the previous value.

Reply:
I think this is due to the backspace bug (#40). Are you able to double-click and then "overwrite" the text? If we have it as placeholder there's no way to just "change" the value (add more info for instance) without writing it all over again.

Reply:
Okay, that makes sense. Okay, I merge it then.